### PR TITLE
Added code for getting file

### PR DESCRIPTION
**Pull request by tzavaliche**

### CODE REVIEW

File: FileStorage.Client/Program.cs [hunk 1]
```diff
@@ -38,9 +38,40 @@ static void Main(string[] args)
...
             Console.ReadLine();
         }
 
+
+        private static void GetFile(string getFile, string saveFile)
+        {
+            HttpWebRequest webRequest = (HttpWebRequest) HttpWebRequest.Create("http://localhost:8081/Api/File/" + getFile);
+            webRequest.Method = "GET";
+
+            (webRequest as WebRequest).Headers.Add(HttpRequestHeader.Authorization,
```
Comment (smurariu) [inline FileStorage.Client/Program.cs]: You don't really need this. It's used when identity server is enabled, I was just playing around and that's why it's there.

File: FileStorage.Client/Program.cs [hunk 2]
```diff
@@ -38,9 +38,40 @@ static void Main(string[] args)
...
+                "Bearer 5c5d3b905c00fdc9817809e324fbe4ab");
+            (webRequest as WebRequest).ContentLength = 0;
```
Comment (smurariu) [inline FileStorage.Client/Program.cs:55]: But what we do need is to add/support the headers from the spec: https://msdn.microsoft.com/en-us/library/azure/dn194274.aspx (see the RequestHeaders section)

File: FileStorage/Controllers/FileController.cs [hunk 3]
```diff
@@ -163,5 +164,53 @@ public HttpResponseMessage CreateFile(string filepath)
...
+                    {
+                        result.StatusCode = System.Net.HttpStatusCode.NotFound;
+                    }
+                    else
+                    {
+                        var stream = new FileStream(Path.Combine("files", filepath), FileMode.Open,
+                                FileAccess.Read, FileShare.None);
+                        result.Content = new StreamContent(stream);
+
+                        result.Headers.Add("Server", System.Environment.MachineName);
```
Comment (smurariu) [inline FileStorage/Controllers/FileController.cs:239]: We should add more headers here: https://msdn.microsoft.com/en-us/library/azure/dn194274.aspx => Response Headers and also I would like us to support ranges

**Comment by tzavaliche** [comment]
@smurariu2 Please have another looksy

**Comment by smurariu** [comment]
Looking good ;)

**Pull request merged**